frontend/src/game/card/CardBase.jsx

```diff
@@ -29,34 +29,37 @@ export default class CardBase extends Component {
   }
 
   getCardImage (side) {
+    const { showFlipped, card } = this.props
     // if initial view is "unflipped" + we want default view (relative to that)
-    if (!this.props.showFlipped && side === DEFAULT) return getCardSrc(this.props.card);
+    if (!showFlipped && side === DEFAULT) return getCardSrc(card);
 
     // if initial view is "flipped" + we want the flipped view (relative to that)
-    if (this.props.showFlipped && side === FLIP) return getCardSrc(this.props.card);
+    if (showFlipped && side === FLIP) return getCardSrc(card);
 
+    // WARNING - this doesn't do anything for kamigawa "flip" cards
+    // the image change there is handled by the class "-rotate"
```

Comment:
This is the gotcha. This who function currently does nothing for cards of layout "flip" it seems
The card data doesn't have a flippedNumber. There is something under "otherIds" and maybe some other urls already attached.

I didn't bother getting into that. Just used the `-rotate` class and took opportunity to add a spinning animattion because why not?